eks-public-cluster.tf

```diff
@@ -56,12 +57,13 @@ module "eks-public" {
       instance_types       = ["t4g.xlarge"]
       capacity_type        = "ON_DEMAND"
       min_size             = 1
-      max_size             = 2 # Allow manual scaling when running operations or upgrades
+      max_size             = local.public_cluster_max_size # Allow manual scaling when running operations or upgrades
       desired_size         = 1
       bootstrap_extra_args = "--kubelet-extra-args '--node-labels=node.kubernetes.io/lifecycle=normal'"
       suspended_processes  = ["AZRebalance"]
       tags = {
-        "k8s.io/cluster-autoscaler/enabled" = false # No autoscaling for these 2 machines
+        "k8s.io/cluster-autoscaler/enabled"                      = true # Autoscaling enabled
```

Comment:
Note for later readers: this enable the autoscaling of the nodepool for this cluster.